### DISCUSSION TITLE
Distance between meter and CT clamps

### DISCUSSION THREAD

**Discussion by mujiace16** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#121] [General]
I understand the CT clamp cable can be extended away form the power meter board. I would like to extend about 50feet with cat 5e cable. Will this exceed the maximum distance between the two?

**Comment by CircuitSetup** [comment]
You should be able to use cat5e wire without issue - you'll just need to calibrate the CT to a known amperage with the wire attached. The longer the wire, the more resistance is introduced. This will show up as less amperage being read by the meter, which is why calibration is necessary.
